lib/agendash.js

```diff
@@ -81,15 +81,23 @@ module.exports = function (agenda, options) {
       /*
       TODO: Need to validate user input.
       */
-      agenda.create(jobName,jobData)
-        .schedule(jobSchedule)
-        .repeatEvery(jobRepeatEvery)
-        .save(function (err) {
-          if (err) {
-            callback('Jobs not created')
-          }
-          callback()
-        });
+      var job = agenda.create(jobName,jobData)
```

Comment:
@vziukas 
I think yours is better.
making an additional remark, this is same thing.
```javascript
if (!jobSchedule && !jobRepeatEvery){
   return callback('Jobs not created')
}
if (jobSchedule){
   job.schedule(jobSchedule)
} 
if (jobRepeatEvery){
   job.repeatEvery(jobRepeatEvery)
} 
```